test/Libraries/PackageManagerTests/RecordedFileSystem.cs

```diff
@@ -63,6 +63,7 @@ public IEnumerable<string> GetDirectories(string dir)
         public void CopyFile(string filePath, string destinationPath)
         {
             this.copiedFiles.Add(new Tuple<string, string>(filePath, destinationPath));
+            this.allFiles.Add(destinationPath);
```

Comment:
This just an improvement to the mock; to make the `allFiles` field represent better the list of all files in the `fileSystem`.